manager/orchestrator/task.go

```diff
@@ -50,7 +50,7 @@ func NewTask(cluster *api.Cluster, service *api.Service, slot uint64, nodeID str
 	}
 
 	// Assign name based on task name schema
-	name := store.TaskName(&task)
+	name := naming.Task(&task)
 	task.Annotations = api.Annotations{Name: name}
```

Comment:
Should we remove this line that stores the task name in raft, as discussed previously?

cc @aluzzardi